packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -18,11 +18,13 @@
   "publisher": "salesforce",
   "license": "BSD-3-Clause",
   "engines": {
-    "vscode": "^1.19.0"
+    "vscode": "^1.21.0"
   },
   "categories": ["Debuggers"],
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "42.10.0",
+    "@salesforce/salesforcedx-apex-debugger": "42.10.0",
```

Comment:
Any estimate on how much work to move to utils and when? This dependency is **really** weird; I suppose I can look the other way for now if you really want to handle this in another PR…

Reply:
Yes, I want to handle this in another PR. There's enough churn with this PR where I really didn't want to add this to the mix.